agent/conf/agent.properties

```diff
@@ -270,3 +270,6 @@ iscsi.session.cleanup.enabled=false
 # This parameter specifies the heartbeat update timeout in ms; The default value is 60000ms (1 min).
 # Depending on the use case, this timeout might need increasing/decreasing.
 # heartbeat.update.timeout=60000
+
+# This parameter specifies if the host must be rebooted when something goes wrong with the heartbeat.
+# reboot.host.and.alert.management.on.heartbeat.timeout=true
```

Comment:
hope the newline is there ;)

Reply:
@DaanHoogland It's there =)

GitHub warns when there is no newline at the end of the file, e.g.:
![image](https://user-images.githubusercontent.com/38945620/127719105-c775eb84-f189-4a89-bdf3-a36c0c678522.png)

Reply:
ah, you're right. forgot about that.